test/tests/functional/pbs_nonprint_characters.py

```diff
@@ -104,6 +104,13 @@ def setUp(self):
             if m.find(msg) != -1:
                 self.n = m.split('=')[0]
                 continue
+        pbs_conf = self.server.pbs_conf
+        if 'PBS_EXEC' in pbs_conf:
+            self.pbs_exec = pbs_conf['PBS_EXEC']
```

Comment:
we are setting self.pbs_exec here, shall we use it for self.qstat_cmd, self.qsmgr_cmd and self.pbsnodes_cmd

Reply:
This is my bad. These lines are not required. They were added while doing some testing and I missed removing them including self.qmgr and self.pbsnodes.